hist/histpainter/src/THistPainter.cxx

```diff
@@ -250,7 +250,7 @@ using `TH1::GetOption`:
 | "BAR"    | Like option "B", but bars can be drawn with a 3D effect.|
 | "HBAR"   | Like option "BAR", but bars are drawn horizontally.|
 | "C"      | Draw a smooth Curve through the histogram bins.|
-| "E0"     | Draw error bars. Markers are drawn for bins with 0 contents.|
+| "E0"     | Draw error bars. Markers are drawn for bins with 0 contents. Combined E1 and E2 it avoid error bars clipping|
```

Comment:
Can you reformulate `Combined E1 and E2 it avoid error bars clipping`? Is it `Combined with E1 and E2`?